controllers/postController.js

```diff
@@ -29,4 +30,74 @@ const upsert = async (req) => {
   );
 };
 
-module.exports = { upsert };
+const list = async (req, res) => {
+  if (!isValidString(req.params.keywordId) || isBlank(req.params.keywordId)) {
+    return res.status(400).send({ message: "[InvalidKeywordId] Error occured" });
+  }
+  if (!isValidString(req.query.includedKeyword)) {
+    return res.status(400).send({ message: "[InvalidIncludedKeyword] Error occured" });
+  }
+  if (!isValidNumber(Number(req.query.limit)) || Number(req.query.limit) <= 0) {
+    return res.status(400).send({ message: "[InvalidLimit] Error occured" });
+  }
+  if (!isValidString(req.query.cursorId)) {
+    return res.status(400).send({ message: "[InvalidCursorId] Error occured" });
+  }
+
+  const hasKeywordId = (await keywordModel.findOne({ _id: req.params.keywordId })) !== null;
+  if (!hasKeywordId) {
+    return res.status(400).send({ message: "[NotExistedKeywordId] Error occured" });
+  }
+
+  if (!isBlank(req.query.cursorId)) {
+    const hasCursorId = (await postModel.findOne({ _id: req.query.cursorId })) !== null;
+    if (!hasCursorId) {
+      return res.status(400).send({ message: "[NotExistedCursorId] Error occured" });
+    }
+  }
+
+  const keywordId = req.params.keywordId;
+  const { includedKeyword, cursorId } = req.query;
+  const limit = Number(req.query.limit);
+  let hasNext = false;
+  let postListResult;
+
+  try {
+    if (isBlank(cursorId)) {
+      postListResult = await postModel
+        .find({ keywordId: keywordId })
```

Comment:
```suggestion
        .find({ keywordId })
```
위와 같이 shorthand notation로도 사용 가능하겠습니다.
전달 주신 테스트 URL로도 정상 동작을 확인했는데 검토 부탁 드려요!

Reply:
그렇네요! 수정하겠습니다 👍